server/netty-server/zio/src/main/scala/sttp/tapir/server/netty/zio/ZioUtil.scala

```diff
@@ -0,0 +1,13 @@
+package sttp.tapir.server.netty.zio
+
+import io.netty.channel.{Channel, ChannelFuture}
+import sttp.tapir.server.netty.internal.FutureUtil
+import zio.{RIO, ZIO}
+
+object ZioUtil {
```

Comment:
this can be private?

Reply:
and this function is available from the parent project?

Reply:
Ah it's a conversion to RIO, ok. Does it need to be public? It looks like an impl detail

Reply:
already moved to private in zio